wpilibc/src/main/native/include/frc/controller/LTVUnicycleController.h

```diff
@@ -82,21 +82,23 @@ class LTVUnicycleController {
    *
    * The reference pose, linear velocity, and angular velocity should come
    * from a Trajectory.
-   * 
-   * The current linear velocity of the chassis can be found from a 
+   *
+   * The current linear velocity of the chassis can be found from a
    * DifferentialDriveWheelSpeeds object using
    * DifferentialDriveWheelSpeeds::ToLinearChassisVelocity.
    *
    * @param currentPose                   The current position of the robot.
-   * @param currentLinearVelocity         The current linear velocity of the robot.
-   *                                      this can be determined by averaging the
-   *                                      measured left and right wheel velocities.
+   * @param currentLinearVelocity         The current linear velocity of the
+   * robot. this can be determined by averaging the measured left and right
```

Comment:
Each comment's continuation lines should be indented to the same level as the first line.

Reply:
I think wpiformat changed the indentation of the continuation lines, should I change nit back?

Reply:
If it changed the indentation, make continuations match the new indentation of the first line for the given @param.